### ISSUE TITLE
TCPv4 counters not appearing in the metrics endpoint

### ISSUE THREAD

**Issue by rgl** [leoluk/perflib_exporter#8]
I'm trying to see the TCPv4 metrics, but for some reason they are not exported. I'm running the exporter as (running with default settings also has the same effect):

```
.\perflib_exporter.exe --perflib.objects=638 --log.level=debug
```

I can see the counters on the log:

```
DEBU[0086] multi counter TCPv4 ->  -> Segments/sec       source="collector.go:109"
DEBU[0086] multi counter TCPv4 ->  -> Connections Established  source="collector.go:109"
DEBU[0086] multi counter TCPv4 ->  -> Connections Active  source="collector.go:109"
DEBU[0086] multi counter TCPv4 ->  -> Connections Passive  source="collector.go:109"
DEBU[0026] OK: perflib collector succeeded after 0.038999s.  source="exporter.go:81"
```

But they never appear on the `/metrics` endpoint (e.g. there is no metric with `connections_established` in its name).

Though, doing the same on the dump endpoint, http://localhost:9432/dump?query=638, the values are displayed:

Name | Mangled name | Type | IsCounter | IsNsCtr | Value | Help Text 
-- | -- | -- | -- | -- | -- | --
[642] Connections Established | connections_established | 0x10000 | false | false | 25 | Connections Established is the number of TCP connections for which the current state is either ESTABLISHED or CLOSE-WAIT.

Is there anything else I can do to troubleshoot this?

**Comment by leoluk** [comment]
See #10 - we need to figure out how to deal with multi value counters

**Comment by bismarck** [comment]
This isn't actually a multi value counter issue.  The problem is that some metrics are being misidentified as multi value counters because counter definition indices are not unique across objects.  

For example,  if you request the `LogicalDisk` provider, you're also going to get it's dependency the `PhysicalDisk` provider.  Both providers have a `current_disk_queue_length` counter with an index of 198.  As a result, the `current_disk_queue_length` counter gets filtered out.

The same thing occurs with `connections_established` which exists in `TCPv4` and `TCPv6` providers as index 642.

**Comment by leoluk** [comment]
Fixed by #13 (thanks @bismarck)
